brevo.com.domain-authentication.json

```diff
@@ -3,24 +3,41 @@
   "providerName": "Brevo",
   "serviceId": "domain-authentication",
   "serviceName": "Brevo Domain Authentication",
-  "version": 2,
+  "version": 3,
   "syncPubKeyDomain": "brevo.com",
   "syncRedirectDomain": "app.brevo.com",
   "logoUrl": "https://static.brevo.com/meetings/assets/brevo_logo.png",
   "description": "Places TXT record for domain verification and DKIM records to authenticate email sent by Brevo on behalf of the user",
-  "variableDescription": "The variable %hash% represents a MD5 hash. %dkim_selector% is the DKIM selector for placing the DKIM public key. This needs to be a variable because the selector is dynamically generated. %dkim_public_key% is the DKIM public key.",
+  "variableDescription": "The variable %hash% represents a combination of prefix string and a MD5 hash. %dkim_selector% is the DKIM selector for placing the DKIM public key. This needs to be a variable because the selector is dynamically generated. %dkim_public_key% is the DKIM public key. %dmarc_value% is dmarc value. %dkimcname% is cname value where DKIM will point",
   "records": [
     {
-        "type": "TXT",
-        "host": "@",
-        "data": "brevo-code:%hash%",
-        "ttl": 3600
+      "groupId": "code",
+      "type": "TXT",
+      "host": "@",
+      "data": "%hash%",
+      "ttl": 3600
     },
     {
-        "type": "TXT",
-        "host": "%dkim_selector%._domainkey",
-        "data": "%dkim_public_key%",
-        "ttl": 3600
+      "groupId": "dkim_raw",
+      "type": "TXT",
+      "host": "%dkim_selector%._domainkey",
+      "data": "%dkim_public_key%",
+      "ttl": 3600
+    },
+    {
+      "groupId": "dmarc",
```

Comment:
As DMARC tends to rise a need to be edited afterwards manually there is quite useful to define this record as non essential. Just for consideration.

Reply:
Thanks for suggestion. Done